### PR TITLE
Vendor gardener/gardener@v1.19.0

### PR DESCRIPTION
**Pull request by stoyanr**
**How to categorize this PR?**

/area open-source
/kind enhancement
/priority 3
/platform kubevirt

**What this PR does / why we need it**:

* Vendor gardener/gardener@v1.19.0
* Update `machine-controller-manager` image
* Update images in `Dockerfile`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
`github.com/gardener/gardener` dependency is now updated to `v1.19.0`. For the complete list of changes, see the [release notes](https://github.com/gardener/gardener/releases/tag/v1.19.0).
```

### CODE REVIEW

**Comment by stoyanr** [comment]
Code generation in CI fails with:
```
../../../../vendor/k8s.io/kube-openapi/pkg/util/proto/document.go:24:2: cannot find package "github.com/googleapis/gnostic/openapiv2" in any of:
	/go/src/github.com/gardener/gardener-extension-provider-kubevirt/vendor/github.com/googleapis/gnostic/openapiv2 (vendor tree)
	/usr/local/go/src/github.com/googleapis/gnostic/openapiv2 (from $GOROOT)
	/go/src/github.com/googleapis/gnostic/openapiv2 (from $GOPATH)
...
```

I can't reproduce this locally. Also, there is `github.com/googleapis/gnostic/openapiv2` under vendor. Any idea what might be wrong?

File: pkg/controller/controlplane/add.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ type AddOptions struct {
 func AddToManagerWithOptions(mgr manager.Manager, opts AddOptions) error {
 	return controlplane.Add(mgr, controlplane.AddArgs{
 		Actuator: genericactuator.NewActuator(kubevirt.Name, controlPlaneSecrets, nil, configChart, controlPlaneChart, controlPlaneShootChart,
-			storageClassChart, nil, NewValuesProvider(logger), extensionscontroller.ChartRendererFactoryFunc(util.NewChartRendererForShoot),
+			storageClassChart, nil, nil, NewValuesProvider(logger), extensionscontroller.ChartRendererFactoryFunc(util.NewChartRendererForShoot),
```
[the remark this answers is not in the document]
Comment (stoyanr) [reply pkg/controller/controlplane/add.go]: Thanks! Fixed.

**Comment by ialidzhikov** [comment]
Try:
```
$ git mv vendor/github.com/googleapis/gnostic/OpenAPIv2 vendor/github.com/googleapis/gnostic/OpenAPIv2-temp
$ git mv vendor/github.com/googleapis/gnostic/OpenAPIv2-temp vendor/github.com/googleapis/gnostic/openapiv2
```

**Comment by stoyanr** [comment]
@ialidzhikov Yes, this solved it, thanks a lot!

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**